File: lua/1132.lua
local x = io.read("*n")
local y = io.read("*n")
local sum = 0

if y < x then
    x, y = y, x
end

for i = x, y do
    if i % 13 ~= 0 then
      sum = sum + i
    end
end

print(sum)

File: go/1132.go
package main

import "fmt"

func main() {
  var x, y, sum int
  fmt.Scanf("%d", &x)
  fmt.Scanf("%d", &y)

  if y < x {
    x, y = y, x 
  }

  for i := x; i <= y; i++ {
    if i % 13 == 0 {
      continue
    }

    sum += i
  }

  fmt.Println(sum)
}
